Automattic/woocommerce-payments
# Allow PO accounts to continue the first part of onboarding

Requires https://github.com/Automattic/woocommerce-payments-server/pull/4172

Fixes #7065 

#### Changes proposed in this Pull Request

* Passes a parameter to identify that a request comes from a progressive onboarding without details submitted (before the first part of PO KYC Is completed, so partially onboarded) to allow the generated link to not collect payments at that moment in time, allowing the user to continue with the standard PO flow.


#### Testing instructions

* Make sure you have checkout https://github.com/Automattic/woocommerce-payments/pull/7716 in your local server
* Enable the new onboarding flow using Dev Tools
* Onboard a new test account setting "United States", "Individual", "Less than $250k" and "My store is already live" in order to trigger the PO account creation
- Leave the onboarding at any step of the first KYC in Stripe. You can leave on the first one, and you'll be redirected to Overview seeing a task to continue setting up payments.
- When clicking on "Set up payments" in the task, you'll be redirected to the step that you abandoned the flow before. If you continue and exit again, you'll be redirected back to the right step.
- The KYC you see, after returning, is still the short version without having to set up deposits at that time (is left as optional before clicking on "Submit")
- Once you submit the details you are redirected to Overview, your account has "Payments" enabled, "Deposits" as "Pending verification" and the status in "Account details" is "Pending".
- To verify the full lifecycle of a PO account, you can set up deposits and see everything as Enabled.

-------------------


**Post merge**
